rumbl/web/controllers/auth.ex

```diff
@@ -8,22 +8,33 @@ defmodule Rumbl.Auth do
 
   def call(conn, repo) do
     user_id = get_session(conn, :user_id)
+
     cond do
       user = conn.assigns[:current_user] -> conn
+        put_current_user(conn, user)
```

Comment:
I'm not sure if I'm just reading this wrong, but don't you need to remove the `conn` at the end of
`user = conn.assigns[:current_user] -> conn` line (or perhaps pipe it in to the `put_current_user` function or something.

Hard to say, I'm not this far in the book yet